package/package.json

```diff
@@ -1,20 +1,17 @@
 {
   "name": "react-native-winrt",
-  "version": "0.0.1",
+  "version": "0.67.0",
```

Comment:
I see a lot of npm packages don't have the RN version in them (And react-native-xaml is at 0.0.62). Should we still be creating separate packages for each major version?

Reply:
Since our version has such a strong tie to the RN(W) version, it seems like this makes the most sense. E.g. in a similar way that `react-native-windows` follows the same major.minor version as `react-native`. Also matches our scheme for Nuget

Reply:
rnx versioning is separate from RNW's (it just happens to be in the same range :))
rnx uses the public API surface for extensibility (winrt modules) whereas react-native-winrt uses turbomodules (which are tightly coupled to RN(W) versions). So agree w/Duncan that you need the strong coupling of version number between rnwinrt and RNW